tests/test_trigger_no_status/results_true.dat

```diff
@@ -1,27 +1,28 @@
 k-combined:
 9.809303E-01 7.264435E-03
-tallies:
+tally 1:
```

Comment:
Personally I'm ok with different order if it means avoiding a class specialization just to preserve the current ordering.